Makefile

```diff
@@ -511,118 +516,116 @@ check-lint: mkdir-reports install-dev		## run linting code style checks
+.PHONY: check-security-only
+check-security-only: mkdir-reports	## run security code checks
 	@echo "Running security code checks..."
 	@-rm -fr "$(REPORTS_DIR)/check-security.txt"
 	@bash -c '$(CONDA_CMD) \
 		bandit -v --ini "$(APP_ROOT)/setup.cfg" -r \
 		1> >(tee "$(REPORTS_DIR)/check-security.txt")'
 
-.PHONY: check-docs
-check-docs: check-doc8 check-docf	## run every code documentation checks
+.PHONY: check-docs-only
+check-docs-only: check-doc8-only check-docf-only	## run every code documentation checks
 
-.PHONY: check-doc8
-check-doc8:	mkdir-reports install-dev		## run PEP8 documentation style checks
+.PHONY: check-doc8-only
+check-doc8-only: mkdir-reports		## run PEP8 documentation style checks
 	@echo "Running PEP8 doc style checks..."
 	@-rm -fr "$(REPORTS_DIR)/check-doc8.txt"
 	@bash -c '$(CONDA_CMD) \
 		doc8 --config "$(APP_ROOT)/setup.cfg" "$(APP_ROOT)/docs" \
 		1> >(tee "$(REPORTS_DIR)/check-doc8.txt")'
 
-# FIXME: move parameters to setup.cfg when implemented (https://github.com/myint/docformatter/issues/10)
-# NOTE: docformatter only reports files with errors on stderr, redirect trace stderr & stdout to file with tee
-# NOTE:
-#	Don't employ '--wrap-descriptions 120' since they *enforce* that length and rearranges format if any word can fit
-#	within remaining space, which often cause big diffs of ugly formatting for no important reason. Instead only check
-#	general formatting operations, and let other linter capture docstrings going over 120 (what we really care about).
-.PHONY: check-docf
-check-docf: mkdir-reports install-dev	## run PEP8 code documentation format checks
+.PHONY: check-docf-only
+check-docf-only: mkdir-reports	## run PEP8 code documentation format checks
 	@echo "Checking PEP8 doc formatting problems..."
 	@-rm -fr "$(REPORTS_DIR)/check-docf.txt"
 	@bash -c '$(CONDA_CMD) \
 		docformatter \
-			--pre-summary-newline \
-			--wrap-descriptions 0 \
-			--wrap-summaries 120 \
-			--make-summary-multi-line \
 			--check \
 			--recursive \
+			--config "$(APP_ROOT)/setup.cfg" \
 			"$(APP_ROOT)" \
 		1>&2 2> >(tee "$(REPORTS_DIR)/check-docf.txt")'
 
-.PHONY: check-links
-check-links: install-dev	## check all external links in documentation for integrity
+.PHONY: check-links-only
+check-links-only:	## check all external links in documentation for integrity
 	@echo "Running link checks on docs..."
 	@bash -c '$(CONDA_CMD) $(MAKE) -C "$(APP_ROOT)/docs" linkcheck'
 
-.PHONY: check-imports
-check-imports: mkdir-reports install-dev	## run imports code checks
+.PHONY: check-imports-only
+check-imports-only: mkdir-reports	## run imports code checks
 	@echo "Running import checks..."
 	@-rm -fr "$(REPORTS_DIR)/check-imports.txt"
 	@bash -c '$(CONDA_CMD) \
 	 	isort --check-only --diff $(APP_ROOT) \
 		1> >(tee "$(REPORTS_DIR)/check-imports.txt")'
 
 .PHONY: check-css
-check-css: mkdir-reports install-npm
+check-css: install-npm check-css-only
+
+.PHONY: check-css-only
+check-css-only: mkdir-reports install-npm
```

Comment:
Should `install-npm` be removed from the `check-css-only` target, only executing it if `check-css` is called?

Reply:
Good catch. Indeed it needs to go.